mllib/src/main/scala/org/apache/spark/mllib/optimization/GradientDescent.scala

```diff
@@ -51,11 +51,9 @@ class GradientDescent private[spark] (private var gradient: Gradient, private va
   }
 
   /**
-   * :: Experimental ::
    * Set fraction of data to be used for each SGD iteration.
    * Default 1.0 (corresponding to deterministic/classical gradient descent)
    */
-  @Experimental
```

Comment:
Please keep this one as experimental as well because it is not implemented efficiently.